cmd/agent/common/helpers.go

```diff
@@ -49,3 +51,34 @@ func setupConfig(confFilePath string, configName string, withoutSecrets bool) er
 	}
 	return nil
 }
+
+// SetupSystemProbeConfig adds the system-probe.yaml file to the config object
+func SetupSystemProbeConfig(sysProbeConfFilePath string) error {
```

Comment:
Do we have to load the entire system-probe yam into viperl to get 1 info ? Would it not be simpler to load the yaml file and return the socket path ?

Reply:
So we wanted to have this ready to read in everything in case people want to add anything into it in the future, especially with the other teams increasingly using the system probe for ebpf.

Reply:
If we want to go this way we won't do it here but add an another config object to the `config` package (just like we have `config.Datadog` we could have `config.SystemProbe`). Anyway like I said in my other comment merging both configuration is error prone. We should either merge system-probe.yaml into datadog.yaml if possible or have two distinct config.

Also, should we move this function into the `pkg/config` package ? It really looks like a config helper that should be run when we load the configuration.

Reply:
I agree with @hush-hush on this.